LiskHQ/lisk-hub
# Add network switcher to splash screen - Closes #1894

### What issue have I solved?

#1894

### How have I implemented/fixed it?

Added network dropdown to splash screen

### How has this been tested?

1. Go to splash screen
2. Change the network in the header
3. Go to the login page and try to log in
4. Check the network you are on.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
